snippets/colorize.md

```diff
@@ -0,0 +1,35 @@
+### colorize
+
+Add special characters to text to print in color in the console (combined with `console.log()`).
+
+Use template literals and special characters to add the appropriate color code to the string output.
+For background colors, add a special character that resets the background color at the end of the string.
+
+```js
+const colorize = (...args) => (
+  {
```

Comment:
While it will get garbage collected, this will cause a memory space for every log of `colors * string`. In high performance applications this is a bad way of doing it due to the large amount of memory high traffic will consume. Each color or option should be it's own function and it should just be a micro library so that there are no excess memory allocations.